conformance/1.5/training-operator-conformance.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: training-operator-conformance
+  namespace: kf-conformance
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: training-operator-conformance
+  template:
+    metadata:
+      labels:
+        app: training-operator-conformance
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: kf-conformance
+      containers:
+      - name: training-conformance-container
+        image: kubeflow/training-operator-conformance
```

Comment:
Can you add  tag to the image?

Reply:
Done

Reply:
Updated image tag that has new changes https://github.com/kubeflow/training-operator/pull/1787